### DISCUSSION TITLE
Can we configure a prefix or suffix for client provider tables?

### DISCUSSION THREAD

**Discussion by ricardoamm** [Mimetis/Dotmim.Sync#991] [Q&A]
Hello, 

Can we configure a prefix or suffix for client provider tables?
For instance the Remote Table "Table" sincronizes to "[prefix]Table".

Best regards

**Comment by Mimetis** [comment]
`TrackingTablesPrefix  `& `TrackingTablesSuffix `?

https://dotmimsync.readthedocs.io/Configuration.html#syncsetup

**Comment by ricardoamm** [comment]
I meant the Local Database Tables. (Remote table A syncs to Local Table prefixA)
I dont have the tracking tables because I'm using the SqlSyncChangeTrackingProvider.

**Reply by Mimetis** [reply]
no, regular tables keeps the same name
